frontend/lib/analytics/google-analytics.tsx

```diff
@@ -285,11 +284,24 @@ type OutboundLinkProps = DetailedHTMLProps<
   href: string;
 };
 
+const defaultOutboundTarget = {
+  target: "_blank",
+};
+
+const defaultOutboundRel = {
+  rel: "noopener noreferrer",
+};
 /**
  * A react component that encapsulates a link to an external website,
  * which we want to track with analytics.
  */
 export function OutboundLink(props: OutboundLinkProps): JSX.Element {
+  if (props.target !== "") {
+    props = { ...props, ...defaultOutboundTarget };
+  }
+  if (props.rel !== "") {
+    props = { ...props, ...defaultOutboundRel };
+  }
```

Comment:
Hmm, I don't _think_ you should have to have these `if` statements in here--what you were doing in 5e5fa43ef1a1ce39f73fb4f8d59d821e6ca22777 worked fine, I think, but your spreads were just in reverse order.

You can think of the spread (`...`) as being syntactic sugar for a `for` loop that basically copies everything from one object to another, which basically means that a spread that goes after another can effectively "override" its values, if that makes sense?

Aside from simpler code, though, another advantage of using your initial implementation (but reversed) is that it would allow for alternative values for `target` and `rel` that aren't just empty strings... for instance, if we wanted `target="boop"` there would actually be no way to achieve that in your current implementation, because the default of `target="_blank"` would always override the `target="boop"` that's passed in.

Since the behavior we want isn't very well-defined, It might help to add a few new test cases here to make it more explicit, such as:

* If we render `<OutboundLink href="blah">` it should result in a link with `target="_blank" rel="noopener noreferrer"`

* If we render `<OutboundLink href="blah" target="boop">` it should result in a link with `target="boop" rel="noopener noreferrer"`

* If we render `<OutboundLink href="blah" target="" referrer="">` it should result in a link with `target="" rel=""`.

* If we render `<OutboundLink href="blah" target={undefined} referrer={undefined}>` it should result in a link without a `target` or `rel``.  (I'm not actually sure if this is possible; I _think_ it is, but I'm interested to see if it's not!)